pkg/controller/operators/catalog/operator.go

```diff
@@ -526,10 +547,20 @@ func (o *Operator) syncObject(obj interface{}) (syncError error) {
 	}
 
 	o.requeueOwners(metaObj)
-
 	return o.triggerInstallPlanRetry(obj)
 }
 
+func (o *Operator) syncOperatorGroup(obj interface{}) (syncError error) {
+	// Assert as metav1.Object
```

Comment:
Nit - I don't think this comment adds any value.

Reply:
Thanks! I'll take it out.